src/code_80005FD0.c

```diff
@@ -4736,17 +4736,17 @@ void func_8000DF8C(s32 arg0) {
 GLOBAL_ASM("asm/non_matchings/code_80005FD0/func_8000DF8C.s")
 #endif
 
-s32 func_8000ED14(s32 arg0, s16 arg1) {
+s32 func_8000ED14(s32 actorIndex, s16 arg1) {
     s32 i;
     s32 a2 = 0;
 
     for (i = 0; i < 8; i++) {
 
-        if (D_801641F8[i].unkC == 0) {
-            D_801641F8[i].unkC = 1;
-            D_801641F8[i].unkE = arg0;
-            D_801641F8[i].unk10 = arg1;
-            D_801641F8[i].unk14 = 0;
+        if (gUnexpiredActorsList[i].unkC == 0) {
+            gUnexpiredActorsList[i].unkC = 1;
+            gUnexpiredActorsList[i].actorIndex = actorIndex;
+            gUnexpiredActorsList[i].unk10 = arg1;
+            gUnexpiredActorsList[i].unk14 = 0;
```

Comment:
Given that there's an actual actor list. Could this be gUnexpiredShellList?

(Don't worry too much about this one, it could always get renamed later if it's not 100% correct)